app.js

```diff
@@ -13,6 +13,61 @@ app.use(express.static(__dirname + '/public'));
 
 // setting the spotify-api goes here:
 
+const spotifyApi = new SpotifyWebApi({
+    clientId: process.env.CLIENT_ID,
+    clientSecret: process.env.CLIENT_SECRET
+  });
+
+// Retrieve an access token
+spotifyApi
+  .clientCredentialsGrant()
+  .then(data => spotifyApi.setAccessToken(data.body['access_token']))
+  .catch(error => console.log('Something went wrong when retrieving an access token', error));
 // Our routes go here:
 
+app.get('/', (req, res) => {
+    res.render('home');
+  });
+
+  // search by query (artist)
+  app.get('/artist-search', (req, res) => {
+    spotifyApi
+  .searchArtists(req.query.artist)
+  .then(async data => {
+    console.log('Hola: ', data.body.artists.items[0]);
+    res.render('artist-search-result', { artists: data.body.artists.items });
+  })
+  .catch(err => console.log('The error while searching artists occurred: ', err));
+  });
+
+
+  // search by id (albums)
+
+  app.get('/albums/:artistId', (req, res, next) => {
+
+    spotifyApi
+    .getArtistAlbums(req.params.artistId)
+    .then(async (data) => {
```

Comment:
why the async function here? 
this is a promise's .then callback -> it doesn't really need to be an async function + you are not even using await inside the function either, so async makes 0 sense at this place tbh.